packages/react/src/components/TimePicker/TimePickerDropdown.jsx

```diff
@@ -183,6 +186,12 @@ const TimePickerDropdown = ({
     }
   };
 
+  const handleOnKeyUp = (e) => {
+    if (e.key === 'Enter') {
+      setOpenState(true);
+    }
+  };
+
```

Comment:
This should not be applied to this component as it breaks the ability for [implicit submission](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#implicit-submission). The icon is what allows for the dropdown functionality on enter and is separate tab stop than the text input.

Reply:
> This should not be applied to this component as it breaks the ability for [implicit submission](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#implicit-submission). The icon is what allows for the dropdown functionality on enter and is separate tab stop than the text input.

There is a bug in <DateTimePickerV2>. The keyup event can detect the enter key but the keydown event did not detect.